include/boost/geometry/algorithms/detail/overlay/get_turn_info_la.hpp

```diff
@@ -261,7 +263,7 @@ struct get_turn_info_linear_areal
                         AssignPolicy::apply(tp, pi, qi, inters);
                         
                         // conditionally handle spikes
-                        if ( ! handle_spikes
+                        if ( ! BOOST_GEOMETRY_CONDITION(handle_spikes)
```

Comment:
I don't object to extra spaces in multi line expressions as this makes the layout quite clear here